src/unlimited.cpp

```diff
@@ -1153,7 +1153,16 @@ bool IsChainNearlySyncd()
     LOCK(cs_ischainnearlysyncd);
     return fIsChainNearlySyncd;
 }
-
+uint64_t LargestBlockSeen(uint64_t nBlockSize)
+{
+     // return the largest block size that we have seen since startup
+     static uint64_t nLargestBlockSeen = BLOCKSTREAM_CORE_MAX_BLOCK_SIZE;
```

Comment:
We don't claim the work on Core is "owned" by Blockstream.
We believe many Core's developers are owners, owned or hired by Blockstream. And this gives Blockstream (AKA you) the ability to hijack the development of Core in a direction favorable only to Blockstream's interests.
We believe we are entitled to our beliefs.
We believe  we are entitled to our freedom of speech and writing, as you are.

Would you prefer "BULLSHIT_CO_MAX_BLOCK_SIZE"? 
This would refer more to your attitude than your company.

BTW, I don't believe the use of "BLOCKSTREAM_CORE_MAX_BLOCK_SIZE" as a name of a variable inside a GitHub repository infringe any of your rights.